src/components/AdvancedFacetViewModal/style.css

```diff
@@ -1,32 +1,10 @@
-.advancedFacetsOverlay {
-  position: absolute;
-  left: 0px;
-  right: 0px;
-  top: 0px;
-  bottom: 0px;
-  background: rgba(0, 0, 0, 0.5);
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  z-index: 1000;
-}
-.advacnedFacetsContainer {
-  width: 1000px;
-  height: 800px;
-  position: relative;
-  background: white;
-  border-radius: 5px;
-  display: flex;
-  flex-direction: column;
-  overflow: hidden;
-}
-.advacnedFacetsContainer .advancedFacetsWrapper {
+.advancedFacetsWrapper {
   flex: 1;
   display: flex;
-  padding: 20px;
-  padding-bottom: 0px;
+  min-height: 700px;
+  max-height: 800px;
 }
-.advacnedFacetsContainer .advancedFacetsTitle {
+.advancedFacetsTitle {
```

Comment:
wait these classes aren't still used because you deleted it here, https://github.com/kids-first/kf-portal-ui/pull/98/files#diff-81ef1d93b3acd5aa96fc27e3be1bd3a3L34 right? And now using https://github.com/kids-first/kf-portal-ui/pull/98/files#diff-b17c47ef8137161ae8a178243850f1c4R12 for the header style?